content-service/src/main/java/io/meeds/news/service/impl/NewsServiceImpl.java

```diff
@@ -1487,6 +1586,11 @@ private News buildArticle(String newsId) throws WikiException {
         if (properties.containsKey(NEWS_VIEWS) && StringUtils.isNotEmpty(properties.get(NEWS_VIEWS))) {
           news.setViewsCount(Long.parseLong(properties.get(NEWS_VIEWS)));
         }
+        if (properties.containsKey(NEWS_ACTIVITY_POSTED)) {
+          news.setActivityPosted(Boolean.valueOf(properties.get(NEWS_ACTIVITY_POSTED)));
```

Comment:
I think that this property is stored inverted, now if I check the "post in activity stream" option when creating the article, the related activity is hidden, and vice versa

Reply:
fixed!